.changeset/empty-grapes-care.md

```diff
@@ -3,7 +3,6 @@
 '@graphql-codegen/typescript-operations': patch
 '@graphql-codegen/typescript': patch
 '@graphql-codegen/typescript-resolvers': patch
-'@graphql-codegen/cli': patch
```

Comment:
I don't think the CLI package needs to be bumped in this case because only the test was changed and it didn't have `@graphql-codegen/visitor-plugin-common` or `@graphql-codegen/cli` in its dep list